docs/enable-implicit-provisioning.adoc

```diff
@@ -0,0 +1,76 @@
+// Copy of this topic: https://docs.ota.here.com/prod/enable-implicit-provisioning.html adapted for aktualizr repo. Replaces duplicate "implicit-provisioning.adoc".
```

Comment:
This file should get renamed, right?

Reply:
Yes, It did get renamed. I'm confused. What is "still a problem"?
The renamed file is [here](https://github.com/advancedtelematic/aktualizr/blob/doc/OTA-2606/rename-provisioning/docs/provision-with-device-credentials.adoc).

Reply:
My bad, I screwed that up somehow.